lib/riax.ex

```diff
@@ -1,81 +1,192 @@
 defmodule Riax do
   @moduledoc """
-  Module to superficially interact with the implemented VNodes, and
-  converage commands.
-  As the implemented VNode is mostly a Key-Value store (for now)
-  this module reflects that.
+  Module to interact with the VNode module given
+  to the Riax.Supervisor module
   """
 
   @doc """
-  Store a value tied to a key
+  Prints the [ring status](https://github.com/basho/riak_core/wiki#ring).
+  The ring is, basically, a representation of the partitioned keys over nodes.
+  Here's a [visual representation](https://github.com/lambdaclass/riak_core_tutorial/blob/master/ring.png)
+  of said ring
+  ## Example:
+  Join 2 running nodes and print the ring, to see the key
+  distribution (handoff) result:
+  ```
+  iex(dev2@127.0.0.1)3> Riax.ring_status
+    ==================================== Nodes ====================================
+    Node a: 64 (100.0%) dev2@127.0.0.1
+    ==================================== Ring =====================================
+    aaaa|aaaa|aaaa|aaaa|aaaa|aaaa|aaaa|aaaa|aaaa|aaaa|aaaa|aaaa|aaaa|aaaa|aaaa|aaaa|
+    :ok
+  iex(dev2@127.0.0.1)13> Riax.join('dev1@127.0.0.1')
+    13:51:21.258 [debug] Handoff starting with target: {:hinted, {913438523331814323877303020447676887284957839360, :"dev1@127.0.0.1"}}
+    ...
+  iex(dev2@127.0.0.1)6> Riax.ring_status
+    ==================================== Nodes ====================================
+    Node a: 64 (100.0%) dev1@127.0.0.1
+    Node b: 0 (  0.0%) dev2@127.0.0.1
+    ...
+  iex(dev2@127.0.0.1)11> Riax.ring_status
+    ==================================== Nodes ====================================
```

Comment:
it would be good to mention that you re-run the same command after a (little) while to see the new distribution

Reply:
Done!